### PR TITLE
Update MPSCNN code to the latest API

### PR DESCRIPTION
**Pull request by s1ddok**
This fixes code in mpscnn that wasn't compiling before.

However I had to comment-out some tests because some of them don't have corresponding CPU kernel, correct me If I searched bad.

### CODE REVIEW

**Comment by Yangqing** [comment]
@ajtulloch Andrew - would you mind taking a look?

File: caffe2/contrib/mpscnn-fb/mpscnn_test.mm [hunk 1]
```diff
@@ -11,6 +11,7 @@
 #include "caffe2/core/workspace.h"
 #include "caffe2/utils/proto_utils.h"
 
+#define CHECK_NEAR(A, B, C) assert(abs(A-B) < C);
```
Comment (ajtulloch) [inline caffe2/contrib/mpscnn-fb/mpscnn_test.mm:14]: This comes from `glog`, FWIW. https://github.com/google/glog/blob/e56ed5520846cbd893944dccfbb1042d187714a6/src/glog/logging.h.in#L842

What version of glog were you using?

File: caffe2/contrib/mpscnn-fb/mpscnn_test.mm [hunk 2]
```diff
@@ -655,7 +656,7 @@ void testMPSCNN() {
     }
   }
 
-  {
+  /*{
```
Comment (ajtulloch) [inline caffe2/contrib/mpscnn-fb/mpscnn_test.mm:659]: IMO it's a bit cleaner to have just `if (CPUOperatorRegistry()->Has(<optype>) { ...}`, but it's on my list to refactor all of this so not a big deal.

**Review by ajtulloch — APPROVED** [review]
LGTM. I'll import and land this, and incorporate into the internal mobile repo next time we sync. Thanks!

**Comment by Yangqing** [comment]
This is now obsolete with our recent ios sync - @s1ddok if you'd like to kindly give the new contrib/ios a shot that would be great :)

**Pull request closed without merging**